integrations/operator/Dockerfile.gha

```diff
@@ -0,0 +1,105 @@
+ARG BASE_IMAGE=gcr.io/distroless/cc-debian12
+
+# BUILDPLATFORM is provided by Docker/buildx
+FROM --platform=$BUILDPLATFORM docker.io/debian:12 as builder
+ARG BUILDARCH
+
+## Install dependencies.
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
+    ca-certificates \
+    unzip \
+    # x86_64 dependencies and build tools
+    build-essential \
+    # ARM dependencies
+    libc6-dev-armhf-cross \
+    gcc-arm-linux-gnueabihf \
+    # ARM64 dependencies
+    libc6-dev-arm64-cross \
+    gcc-aarch64-linux-gnu \
+    # i386 dependencies
+    libc6-dev-i386-cross \
+    gcc-i686-linux-gnu
+
+# Create a basic Makefile in /tmp/v from build.assets/versions.mk to print
+# the program versions we need.
+RUN mkdir -p /tmp/v
+COPY build.assets/versions.mk /tmp/v
+COPY <<-EOT /tmp/v/Makefile
+  include versions.mk
+  print-go-version:; @echo $(or $(GOLANG_VERSION),$(error GOLANG_VERSION not defined))
+  print-protoc-version:; @echo $(or $(PROTOC_VERSION),$(error PROTOC_VERSION not defined))
+EOT
+
+# Install Go.
+RUN GOLANG_VERSION=$(make -s -C /tmp/v print-go-version) && \
+    mkdir -p /opt && \
+    curl -fsSL https://storage.googleapis.com/golang/$GOLANG_VERSION.linux-${BUILDARCH}.tar.gz | \
+      tar xz -C /opt && \
+    chmod a+w /var/lib && \
+    chmod a-w /
+
+ENV GOPATH="/go" \
```

Comment:
Just curious. How hard is it to do a separate FIPS-specific build of the operator?

Reply:
Oh, it's way easier than that. Just need to set `GOEXPERIMENT=boringcrypto` as an environment variable before building. That's it.

Reply:
I ask because of https://github.com/gravitational/teleport/issues/24422. Was just curious how much work it would be since you're redoing this anyway.

Reply:
Seems quite doable then, but not as part of the effort to migrate from Drone to GHA. We just need to get off drone first which is broad enough without other distractions. But we can look at it soon.

Reply:
Yup, not trying to add work. Just was curious. :-)